src/ekf.cpp

```diff
@@ -204,6 +205,10 @@ namespace RobotLocalization
                "\nCorrected full estimate error covariance is:\n" << estimateErrorCovariance_ <<
                "\n\n---------------------- /Ekf::correct ----------------------\n");
     }
+    else
+    {
+      ROS_WARN("Rejecting measurement due to Mahalanobis Threshold");
```

Comment:
I think this check should be here: https://github.com/clearpathrobotics/robot_localization/blob/88f8b565430e890de5f20f368fb9210f55ee5ede/src/filter_base.cpp#L393

So it happens for any filter using this: EFK, UKF or any other.